uber/models.py

```diff
@@ -933,6 +933,18 @@ def slots_taken(self):
         return len(self.shifts)
 
     @property
```

Comment:
These seem like generally useful properties which definitely made this implementation read more cleanly.